training/src/app/main/main.component.scss

```diff
@@ -0,0 +1,147 @@
+#fire {
+  position: relative;
+  width: 1126px;
+  height: 521px;
+  left: 157px;
+  top: -17px;
+}
+
+#chuck-pic {
+  position: absolute;
+  width: 283px;
+  height: 285px;
+  left: 576px;
+  top: 329px;
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.11));
+  border-radius: 142.5px;
+}
+
+#chuck-norris{
+  position: absolute;
+  width: 349px;
+  height: 66px;
+  left: 545px;
+  top: 503px;
+}
+.parent {
+  position: relative;
+  top: 0;
+  left: 0;
+  width: fit-content;
+  block-size: fit-content;
+}
+
+.rectangle-1{
+  position: absolute;
+  width: 912px;
+  height: 124px;
+  left: 264px;
+  top: 724px;
+
+  background: #FEF2ED;
+}
+
+#go-button{
+  position: absolute;
+  width: 141.24px;
+  height: 39px;
+  left: 996px;
+  top: 766px;
+
+  background: #F05924;
```

Comment:
lets move colors to the style.scss files
for example:
$orange-color: #F05924;

and then u can simply put $orange-color instead of the color itself.
this way if u need to change it, u wont have to go to many places and change, just one place